charts/burrow/templates/burrow-configmap.yaml

```diff
@@ -27,18 +27,27 @@ data:
     [client-profile.{{ $key }}]
     kafka-version={{ $value.kafkaVersion | default "0.8" | quote}}
     client-id="{{ $value.clientId | default "burrow-lagchecker" }}"
+    tls="{{ $value.tls | default "burrow-mTLS" }}"
```

Comment:
@fktkrt can you please make `tls=` block conditionally inserted. The default should be empty as mentioned in the docs https://github.com/linkedin/Burrow/wiki/Configuration#client-profile

Reply:
Yes. it's done.